Automated Smoketest: :x:
OS: centos 7.6.1810 (Linux 3.10.0-957.1.3.el7.x86_64)
 GCC: gcc (GCC) 7.3.1 20180303 (Red Hat 7.3.1-5)
 Host: ip-10-20-0-17.ca-central-1.compute.internal
Sha: c7299f80ef0fab22cca0c50673d5812147ff950d
Number of Error(s): 3
make[2]: *** [esp/src/CMakeFiles/eclwatch.dir/build.make:303: /home/centos/smoketest/PR-13341/HPCC-Platform/esp/src/build/dist/stub.eclwatch.js] Error 1
make[1]: *** [CMakeFiles/Makefile2:10374: esp/src/CMakeFiles/eclwatch.dir/all] Error 2
make: *** [Makefile:163: all] Error 2

[Build error:](https://gist.github.com/25837c3781ce8ae72cef6ba8c3fcb06a)
Build log [Build log](https://gist.github.com/HPCCSmoketest/25837c3781ce8ae72cef6ba8c3fcb06a#file-result-20-02-20-14-55-20-log)

--- a/esp/src/src/react/theme.ts
+++ b/esp/src/src/react/theme.ts
@@ -1,0 +1,28 @@
+import { createMuiTheme } from "@material-ui/core/styles";
+
+export const theme = createMuiTheme({
+    palette: {
+      primary: {
+          main: "#1a9bd7",
+          light: "#66ccff",
+          dark: "#006da5",
+          contrastText: "#f5f5f5"
+      },
+      secondary: {
+          main: "#455a64",
+          light: "#718792",
+          dark: "#1c313a",
+          contrastText: "#fff"
+      }
+    },
+    overrides: {  // lets make everything look the same. if we want to overwrite use the useStyles hook inside the component
+        MuiStepIcon: {
+            root: {
+                color: "rgba(0, 0, 0, 0.38)"
+            },
+            completed: {
+                color: "#1a9bd7"
+            }
+        }
+    }
+});

--- a/esp/src/src/react/wuStatus.tsx
+++ b/esp/src/src/react/wuStatus.tsx
@@ -2,6 +2,8 @@ import { Workunit, WUStateID } from "@hpcc-js/comms";
 import Step from "@material-ui/core/Step";
 import StepLabel from "@material-ui/core/StepLabel";
 import Stepper from "@material-ui/core/Stepper";
+import { MuiThemeProvider  } from "@material-ui/core/styles";
+import { theme } from './theme';
 import * as React from "react";
 
 import "dojo/i18n";
@@ -88,15 +90,19 @@ export const WUStatus: React.FunctionComponent<WUStatus> = ({
         };
     }, []);
 
-    return <Stepper activeStep={activeStep} alternativeLabel>
-        {steps.map((step, i) => {
-            const labelProps = {
-                error: i <= activeStep ? failed : false
-            };
-            const label = activeStep === i ? step.activeText : step.text;
-            return <Step key={i}>
-                <StepLabel {...labelProps}>{label}</StepLabel>
-            </Step>;
-        })}
-    </Stepper>;
+    return (
+        <MuiThemeProvider theme={theme}>
+            <Stepper activeStep={activeStep} alternativeLabel>
+                {steps.map((step, i) => {
+                    const labelProps = {
+                        error: i <= activeStep ? failed : false
+                    };
+                    const label = activeStep === i ? step.activeText : step.text;
+                    return <Step key={i}>
+                        <StepLabel {...labelProps}>{label}</StepLabel>
+                    </Step>;
+                })}
+            </Stepper>;
+        </MuiThemeProvider>
+    )
 };

